pkg/intermediate/aggregate.go

```diff
@@ -168,9 +169,23 @@ func (a *AggregationProcess) addOrUpdateRecordInMap(flowKey *FlowKey, record ent
 	a.mutex.Lock()
 	defer a.mutex.Unlock()
 
+	correlationRequired := false
+	// Get Antrea flowType from the record that will help to determine if correlation
+	// is required or not.
+	if ieWithValue, exist := record.GetInfoElementWithValue("flowType"); exist {
+		if ieWithValue.Element.DataType == entities.Unsigned8 {
+			recordFlowType := ieWithValue.Value.(uint8)
+			// Correlation is required for only InterNode flow type defined in
+			// pkg/registry/registry.go.
+			if recordFlowType == uint8(registry.InterNode) {
```

Comment:
Maybe we can define AntreaFlowType constants to be like `registry.InterNode AntreaFlowType = uint8(1)` to avoid conversion here.

Reply:
Removed the type AntreaFlowType completely to make it simpler.